content/rule_book/roles_and_responsibilities.md

```diff
@@ -48,14 +48,14 @@ The Quizmaster must:
 
 - All conferring among quizmaster and answer judge(s) shall be done privately.
     - It is the answer judge who will determine the accuracy of its content and make a ruling accordingly.
-- Will watch the quizmaster's questions for accuracy of reading.
+- Will watch the quizmaster's questions for accuracy of reading the question.
```

Comment:
hmmmm, technically adding "the question" excludes the AJ watching the QM's "question number five, question" for accuracy.

Reply:
I added the question so that the proposition "of reading" is attached to something. Currently "of reading" doesn't make sense. Prepositions should be able to be removed and the sentence still make sense, even if vague. If you remove "for accuracy" the meaning of "Will watch the quizmaster's questions of reading" doesn't make sense.

Reply:
"Will watch the quizmaster's reading for accuracy" maybe??

Reply:
I like that wording.